notebooks-cpd/Event_Store_Data_Feed.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data feed (optional)\n",
+    "\n",
+    "Using a notebook to feed data into Event Store is now optional for this code pattern.\n",
+    "Instead, you should use IBM Streams to feed the data. If you are not using IBM Streams,\n",
+    "you can run this notebook as an alternative.\n",
+    "\n",
+    "In this notebook, a CSV file is read into a pandas DataFrame\n",
+    "which is then used to batch insert the rows into Event Store.\n",
+    "\n",
+    "Before running this notebook, you need to run the **Event Store Table Creation**\n",
+    "notebook to setup the database and table."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to IBM Db2 Event Store\n",
+    "\n",
+    "Edit the values in the next cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CONNECTION_ENDPOINT=\"\"\n",
+    "\n",
+    "EVENT_USER_ID=\"\"\n",
+    "\n",
+    "EVENT_PASSWORD=\"\"\n",
+    "\n",
+    "# Port will be 1100 for version 1.1.2 or later (5555 for version 1.1.1)\n",
+    "PORT = \"30370\"\n",
+    "\n",
+    "DEPLOYMENT_ID=\"\"\n",
+    "\n",
+    "# Database name\n",
+    "DB_NAME = \"EVENTDB\"\n",
+    "\n",
+    "# Table name\n",
+    "TABLE_NAME = \"IOT_TEMPERATURE\"\n",
+    "\n",
+    "HOSTNAME=\"\"\n",
+    "\n",
+    "DEPLOYMENT_SPACE=\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "bearerToken=!echo `curl --silent -k -X GET https://{HOSTNAME}:443/v1/preauth/validateAuth -u admin:password |python -c \"import sys, json; print(json.load(sys.stdin)['accessToken'])\"`\n",
```

Comment:
same thing about the password.

Just assume there is no such thing as real enterprise software that people use with a default out-of-the-box user password.  Security matters.  The more you know ;)